public/locale/en.json

```diff
@@ -1014,6 +1022,7 @@
   "file_upload_error": "Error uploading file",
   "file_upload_success": "File uploaded successfully",
   "file_uploaded": "File Uploaded Successfully",
+  "filed": "Filed",
```

Comment:
instead of this, use variables along with the sentence. do not translate each words individually